src/Icon/Icon.jsx

```diff
@@ -20,13 +21,15 @@ export const Icon = ({ name, size, color, rotate, ...props }) => {
   }, [name])
 
   if (typeof name === 'string' && typeof glyphs[name] === 'string') {
-    return <BaseIcon
-      path={glyphs[name]}
-      size={size}
-      color={color}
-      spin={rotate}
-      {...props}
-    />
+    return <Flex flexDirection={'row'} justifyContent={'center'} alignItems={'true'}>
```

Comment:
Who do you use a flex with just one child?

Reply:
I use it to center the base icon.